kafka-impl/src/test/java/io/streamnative/pulsar/handlers/kop/coordinator/group/MemberMetadataTest.java

```diff
@@ -13,17 +13,17 @@
  */
 package io.streamnative.pulsar.handlers.kop.coordinator.group;
 
-import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+import static org.testng.internal.junit.ArrayAsserts.assertArrayEquals;
```

Comment:
We can use `assertEquals` for `testMetadata` to eliminate the usage of internal JUnit package.